packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -189,7 +191,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
       onRenderList = this._onRenderList,
       responsiveMode
     } = this.props;
-
+    let { calculatedHeight } = this.state;
```

Comment:
@cschleiden , I have addressed your comment in another PR: https://github.com/OfficeDev/office-ui-fabric-react/pull/1590